### ISSUE TITLE
Back button doesn't work while using high memory consuming apps 

### ISSUE THREAD

**Issue by avinashrocks1990** [LegacyXperia/local_manifests#12]
when using heavy memory consuming apps like games the back button doesn't close the app or minimize it sometimes when you press back button it brings up a previously closed app it continues for 2-3 apps and then the back button stops functioning i have to press home button to get back to the home screen and manually kill the app

**Comment by avinashrocks1990** [comment]
logcat
https://docs.google.com/file/d/0B7z0NoNc5ecUR0w4RFFkeFZFQjQ/edit?usp=sharing

**Comment by amishmm** [comment]
Yes I have noticed once or twice that back button did not work. Dont know what was reason i.e. dont know if some app was using high memory or not.

**Comment by djey47** [comment]
Just experienced it a few minutes ago. I was playing Subway Surfers on xperia neo V (uses a lot of RAM, btw).
Latest build: 20130828

Back button does not work, home button does. Screen reacts to fingers, still.

**Comment by mikeNG** [comment]
Is this still present? I think it was fixed upstream in CM

**Comment by djey47** [comment]
Nope, still present in latest 10.2 after a few minutes of 3D game. 
I could not reproduce it everytime, though.

I used Menu key to get me out of there. Actually works as it is handled by OS, not current app.

**Comment by mikeNG** [comment]
Hmm this seems to be present in cm11 too, soft buttons don't work when the keyboard app gets killed to free up RAM

**Comment by MardonHH** [comment]
As a workaround you can try add an init.d script with this content:

renice -19 'pidof com.android.inputmethod.latin'

**Comment by sean024** [comment]
Hi,

Thanks for the reply,

I came across this page earlier, but i never thought the issue was related,  However, could you explain in detail how to add and change the script or guide me to a link on how to do it.  I'm kind of a newbie in regards to adding the scripts, I've Googled it up, but I can't seem to understand how to. 

Thanks,
